packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -119,6 +129,29 @@ class HotRunner extends ResidentRunner {
 
   DateTime firstBuildTime;
 
+  String _targetPlatform;
+  String _sdkName;
+  bool _emulator;
+
+  Future<void> _calculateTargetPlatform() async {
+    if (_targetPlatform == null) {
```

Comment:
nit, maybe `if (_targetPlatform != null) return;` and then un-indent

Reply:
Done.